trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -168,6 +168,7 @@ describe(page, () => {
     wrapper.expectElement(keyingSheetTabSelector).toExist();
 
     wrapper.expectText(`${keyingSheetTabSelector} h2[data-cy="keying-sheet-heading"]`).toRead('Keying sheet');
+    wrapper.expectText(`${keyingSheetTabSelector} p`).toMatch(/Select payments and mark as done when the adjustments have been keyed into ACBS./);
```

Comment:
Will this assertion break if we add another `p` tag to the page? I think the test might be less brittle if we used the `data-cy` here as the selector.